### PR TITLE
remove trailing spaces from key in cmd_unbind() and cmd_showbind()

### PR DESCRIPTION
**Pull request by jolange**
Resolving minor FIXMEs.

### CODE REVIEW

File: command_mode.c [hunk 1]
```diff
@@ -727,7 +732,12 @@ static void cmd_showbind(char *arg)
 	if (*key == 0)
 		goto err;
 
-	/* FIXME: remove spaces at end */
+	/* remove trailing spaces from key */
+	char *key_end = key + strlen(key);
+	while (isspace(*(key_end-1))) {
+		key_end--;
+	}
+	*key_end = 0;
```
Comment (mahkoh) [inline command_mode.c]: 1. Please no duplicate code.

2. `isspace` is broken. Write a small utility that checks for ' ' and '\t'.

Comment (jolange) [inline command_mode.c]: Both addressed (will squash things later to clean up, if everyone is satisfied).

Just for my own education: what is wrong with `isspace()`?

Comment (mahkoh) [inline command_mode.c]: It accepts only a single byte which is not enough to determine if something is a space. E.g. "a ̈b" contains ' '.

**Comment by flyingmutant** [comment]
@mahkoh, do you mean by "broken isspace" that it does not check for various Unicode (outside ASCII) spaces?

**Comment by mahkoh** [comment]
There are at least three issues:

1. isspace depends on the locale. If a particular locale defines isspace = true for all inputs, then this patch leads to segfaults.

2. I do not know of an official algorithm to determine what a space in unicode is. There are various code points which are referred to as spaces, but those can be combined with other characters so that no space actually appears on the display. Hence the question if something is a space is not well-defined for any unit smaller than grapheme clusters. As I demonstrated above, the ascii space can be combined with  ̊ to form a freestanding  ̊ (usually used to decorate other letters such as å), so this issue is not restricted to code points above 127.

3. isspace is stateless and only accepts single bytes so it cannot be used to detect spaces above code points 127.

**Pull request merged**